e2e/conftest.py

```diff
@@ -179,7 +179,7 @@ def cleanup(self):
             for ev in self.created:
                 requests.delete(api_v0('events/%d' % ev))
             for t in self.teams:
-                re = requests.get(api_v0('events?team=' + t))
+                re = requests.get(api_v0('events?include_subscribed=false&team=' + t))
```

Comment:
if team could have funky characters you could consider using `params={'team': t}` on the requests call